### PR TITLE
fix select query without from

### PR DESCRIPTION
**Pull request by Freax13**
Not all `SELECT` queries require a `FROM` statement. This pr fixes the query generation for queries without `FROM` and allows the following example to work:

```swift
conn.sql()
    .select()
    .column(SQLAlias.init(SQLFunction("LAST_INSERT_ID"), as: SQLIdentifier.init("id")))
    .first()
```

### CODE REVIEW

**Review by gwynne — APPROVED** [review]
Huh, I coulda sworn I'd already fixed that... guess not. Thanks, looks great!

**Review by gwynne — CHANGES_REQUESTED** [review]
@Freax13 Sorry, I just realized two seconds before I hit the merge button... Could you add a unit test for this? 😅

**Review by gwynne — APPROVED** [review]

**Comment by tanner0101** [comment]
These changes are now available in [3.8.1](https://github.com/vapor/sql-kit/releases/tag/3.8.1)

**Pull request merged**
